src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php

```diff
@@ -49,11 +49,12 @@ public function __construct(
     protected function configure(): void
     {
         $this
+            ->addOption('force', 'f', null, 'Force compiling the assets even in debug mode')
             ->setHelp(<<<'EOT'
 The <info>%command.name%</info> command compiles and dumps all the assets in
 the asset mapper into the final public directory (usually <comment>public/assets</comment>).
 
-This command is meant to be run during deployment.
+When run in debug mode, this command actually removes those assets, unless --force is passed.
```

Comment:
```suggestion
When run in debug mode, this command actually removes those assets, unless <inf
o>--force</info> is passed.
```

Reply:
I noticed that we use info vs comment inconsistently in the codebase. I thing info is the most appropriate.
I someone sees this comment, PR welcome :)